app/lib/three_scale/json_representer.rb

```diff
@@ -19,7 +19,7 @@ module ThreeScale::JSONRepresenter
   # This allows only JSON representers to merge XML content from the models
   def to_node(options)
     builder = ThreeScale::XML::Builder.new(skip_instruct: true)
-    doc = options.fetch(:doc)
+    doc = options.fetch(:doc, Nokogiri::XML::Document.new)
```

Comment:
How can I reproduce the error? `key not found: :doc`

If I remove this, I don't see the error

Reply:
These tests were failing (from [here](https://app.circleci.com/pipelines/github/3scale/porta/27737/workflows/f0c595c0-ee6b-47aa-b600-928c92ceab7f/jobs/310670/tests)):

![Screenshot from 2024-04-04 10-23-54](https://github.com/3scale/porta/assets/1270328/451ab1c5-7a0f-40da-b666-d56f1fdae152)
![Screenshot from 2024-04-04 10-24-09](https://github.com/3scale/porta/assets/1270328/f4fd83e8-7b8a-4a36-90c8-9ec568562878)